modules/gateways/xendit/lib/ActionBase.php

```diff
@@ -324,7 +324,7 @@ public function extractCustomer(array $params, bool $isCreditCard = false): arra
             "given_names" => $params['firstname'],
             "surname" => $params['lastname'],
             "email" => $params['email'],
-            "mobile_number" => $params['phonenumber'],
+            "mobile_number" => str_replace(['.', ' '], '', $params['telephoneNumber'])
```

Comment:
What happend if merchant input 08271xxxx?

Reply:
Base our data, this is commend behavior.

Reply:
@candrasaputra 

WHMCS already formatted the phone number. current `phonenumber` will be `0987654321` but `telephoneNumber` formatted `+62987654321` (included country code). So we just need to use `telephoneNumber` for Xendit invoice

Reply:
okay, thanks @andykim